recipe-server/client/control/components/RecipeForm.js

```diff
@@ -121,25 +121,26 @@ export class RecipeForm extends React.Component {
       route,
       routeParams = {},
       recipe = {},
+      revision = {},
       allRevisions = {},
       pristine,
       submitting,
       recipeId,
-      revision,
       user: {
         id: userId,
       },
     } = this.props;
-    const requestDetails = recipe && recipe.approval_request;
+
+    const requestDetails = revision.approval_request;
```

Comment:
I get `Cannot read property 'approval_request' of null` on the Add New Recipe page.